lib/aptible/cli/subcommands/db.rb

```diff
@@ -114,6 +114,24 @@ def self.included(thor)
               render_database(database, database.account)
             end
 
+            desc 'db:replicate SOURCE DEST',
```

Comment:
I think we should try to maintain our naming convention here

```suggestion
            desc 'db:replicate HANDLE REPLICA_HANDLE',
```

or maybe `MASTER_HANDLE` or `SOURCE_HANDLE` for the first arg?

Reply:
Yeah, good point. I matched `aptible db:clone` because it seemed like the most applicable existing operation. If there's another convention though, I'm happy to pattern match on it!

<img width="346" alt="Screen Shot 2019-10-15 at 12 23 55 PM" src="https://user-images.githubusercontent.com/6646098/66854449-f9404a00-ef46-11e9-91b3-2a77f1ee5f6e.png">

Reply:
I think the `SOURCE DEST` is confusing - one might assume you can pass the handle of an existing database as the destination, when this operation actually creates a new database as the result.  I'd argue the closest existing command is `backup:restore`.

`aptible db:replicate HANDLE --replica-handle REPLICA_HANDLE` is a bit verbose in the help text, but yields a clean command in real life, eg : `aptible db:replicate prod-db --replica-handle prod-reporting`

Reply:
In this case I think the REPLICA_HANDLE needs to be a top level item off the subcommand b/c it's a required input. As best I can see using `--replica-handle` would make it optional in the CLI framework we use, which we wouldn't want.

So, I'm just gonna commit josh's change and then we pick back up the convo off that diff if we want.

Reply:
I agree, if we use an optional parameter, we would have to generate a handle in the event that one wasn't supplied. I feel like it would be frustrating to create a replica and find out that it was named incorrectly so the customer would have to deprovision and re-create it or we would have to rename it for them.